src/ui/OmniboxResultList/OmniboxResultList.ts

```diff
@@ -22,68 +22,82 @@ export interface IOmniboxResultListOptions extends IResultListOptions {
 }
 
 /**
- * This component is exactly like a normal ResultList Component, except that it will render itself inside the Omnibox component.
- * This will provide a kind of search as you type functionnality, allowing you to easily render complex Result Templates inside the Omnibox component.
+ * The OmniboxResultList component acts exactly like a normal {@link ResultList} component, except that it renders itself
+ * inside the {@link Omnibox} component.
  *
```

Comment:
I was thinking about mentioning that this component actually inherits from the ResultList component... However, since most of the options don't really seem to apply, I finally decided against it. What do you think?